testsuite/forge/src/success_criteria.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::bail;
+use serde::Serialize;
+use std::time::Duration;
+use transaction_emitter_lib::emitter::stats::TxnStats;
+
+#[derive(Default, Clone, Debug, Serialize)]
+pub struct SuccessCriteria {
+    avg_tps: usize,
+    max_latency_ms: usize,
+}
+
+impl SuccessCriteria {
```

Comment:
I was thinking SuccessCriteria could be something implemented by each experiment, rather than a global config applied for the entire test suite

Reply:
As discussed offline, we can evolve this going forward as we add more experiments support in Forge - I am going to land this change with the current scope for now.